notebooks/cookbook/report_minimization.ipynb

```diff
@@ -0,0 +1,129 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e174828a",
+   "metadata": {},
+   "source": [
+    "## Reporting Minimization\n",
+    "\n",
+    "You can report the status of [`simulation.minimizeEnergy()`](http://docs.openmm.org/development/api-python/generated/openmm.app.simulation.Simulation.html?#openmm.app.simulation.Simulation.minimizeEnergy) using a [`MinimizerReporter`](http://docs.openmm.org/development/api-python/generated/openmm.openmm.MinimizationReporter.html).\n",
+    "\n",
+    "The syntax for doing this is somewhat different to typical OpenMM functionality. Read the [API documentation](http://docs.openmm.org/development/api-python/generated/openmm.openmm.MinimizationReporter.html) for more explanation.\n",
+    "\n",
+    "First we will create a test system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3592a17b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openmm.app import *\n",
+    "from openmm import *\n",
+    "from openmm.unit import *\n",
+    "from sys import stdout\n",
+    "\n",
+    "pdb = PDBFile('villin.pdb')\n",
+    "forcefield = ForceField('amber14-all.xml', 'amber14/tip3pfb.xml')\n",
+    "system = forcefield.createSystem(pdb.topology, nonbondedMethod=PME,\n",
+    "        nonbondedCutoff=1*nanometer, constraints=HBonds)\n",
+    "integrator = LangevinMiddleIntegrator(300*kelvin, 1/picosecond, 0.004*picoseconds)\n",
+    "simulation = Simulation(pdb.topology, system, integrator)\n",
+    "simulation.context.setPositions(pdb.positions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e7b659d",
+   "metadata": {},
+   "source": [
+    "We then create a `MinimizationReporter` class that will print the current energy to the screen every 100 steps and save the energies to an array which we can plot later"
```

Comment:
That should be, "every 100 iterations," not, "every 100 steps."  Does it even take 100 iterations to minimize?  You probably want to print more often.

It would be good to give a little more explanation of what we're doing.  Something like this: "A MinimizationReporter is called after every iteration of minimization.  It is an abstract class.  To use it, you define your own subclass that reports results in whatever way you want.  The reporter shown here prints the current energy to the screen..."